zetaclient/metrics/metrics.go

```diff
@@ -104,6 +104,14 @@ var (
 		Name:      "percentage_of_rate_reached",
 		Help:      "Percentage of the rate limiter rate reached",
 	})
+
+	// SignLatency is a histogram of of the TSS keysign latency
+	SignLatency = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: ZetaClientNamespace,
+		Name:      "sign_latency",
+		Help:      "Histogram of the TSS keysign latency",
+		Buckets:   []float64{1, 15, 30, 60, 120, 240},
```

Comment:
Can you please specify units here? It's not clear whether these are seconds or milliseconds.

UPD: okay, these are seconds. Can we expand values between `1s` and `15s` to support more granular metrics?

Reply:
We could adjust he buckets, but I'm not sure it's particularly useful.